webrender/src/batch.rs

```diff
@@ -1772,16 +1772,16 @@ impl ClipBatcher {
     ) {
         let mut coordinate_system_id = coordinate_system_id;
         for work_item in clips.iter() {
+            let info = clip_store
+                .get_opt(&work_item.clip_sources)
```

Comment:
nit: why `get_opt` if we follow with `expect` right away?

Reply:
Because (for now) we only have a weak handle here, so the freelist API doesn't guarantee it's present, but we know that it should be due to usage here. I intend to refactor how we store clip sources as part of this work.